monai/transforms/transforms.py

```diff
@@ -26,6 +28,81 @@
 export = monai.utils.export("monai.transforms")
 
 
+@export
+class LoadNifti:
+    """
+    Load Nifti format file from provided path.
+    """
+
+    def __init__(self, as_closest_canonical=False, image_only=False, dtype=None):
+        """
+        Args:
+            as_closest_canonical (bool): if True, load the image as closest to canonical axis format.
+            image_only (bool): if True return only the image volume, other return image volume and header dict.
+            dtype (np.dtype, optional): if not None convert the loaded image to this data type.
+        """
+        self.as_closest_canonical = as_closest_canonical
+        self.image_only = image_only
+        self.dtype = dtype
+
+    def __call__(self, filename):
+        """
+        Args:
+            filename (str or file): path to file or file-like object.
+
+        Returns:
+            The loaded image volume if `image_only` is True, or a tuple containing the volume and the Nifti
```

Comment:
This should be moved or included in class or init docstring. During usage, when user writes this transform as part of Compose, they will never see this documentation.

Reply:
Updated in the latest patch.
Thanks.